Common/Securities/Option/QLOptionPriceModel.cs

```diff
@@ -172,15 +183,6 @@ public OptionPriceModelResult Evaluate(Security security, Slice slice, OptionCon
                 // can return negative value in neighborhood of 0
                 var npv = Math.Max(0, EvaluateOption(option));
 
-                // Get time until maturity (in year) and discount factor by dividend and risk free rate
-                var blackVolatilityMaturity = underlyingVol.link.dayCounter()
-                    .yearFraction(underlyingVol.link.referenceDate(), maturityDate);
-                var riskFreeRateMaturity = riskFreeRate.link.dayCounter()
-                    .yearFraction(riskFreeRate.link.referenceDate(), maturityDate);
```

Comment:
`blackVolatilityMaturity` and `riskFreeRateMaturity` have the same value since their calculations depend on `dayCounter` and `referenceDate` which are the same.